xarray/core/indexing.py

```diff
@@ -6,6 +6,7 @@
 from datetime import timedelta
 from typing import Any, Callable, Iterable, List, Optional, Tuple, Union
 
+import dask.array as da
```

Comment:
```suggestion
```
Dask is not a required install so can't just import it. I don't think it is necessary to import though.

Reply:
Oops I should have check all the CI failure not just the first one. Thanks for pointing that out.